pkg/client/apifactory.go

```diff
@@ -28,15 +28,14 @@ import (
 	"github.com/apache/incubator-yunikorn-k8shim/pkg/client/informers/externalversions/yunikorn.apache.org/v1alpha1"
 	"github.com/apache/incubator-yunikorn-k8shim/pkg/common/constants"
 
-	"k8s.io/client-go/informers"
-	"k8s.io/client-go/tools/cache"
-	"k8s.io/kubernetes/pkg/scheduler/volumebinder"
-
-	"github.com/apache/incubator-yunikorn-core/pkg/api"
 	appclient "github.com/apache/incubator-yunikorn-k8shim/pkg/client/clientset/versioned"
 	appinformers "github.com/apache/incubator-yunikorn-k8shim/pkg/client/informers/externalversions"
 	"github.com/apache/incubator-yunikorn-k8shim/pkg/conf"
 	"github.com/apache/incubator-yunikorn-k8shim/pkg/log"
+	apiCommon "github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/api"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kubernetes/pkg/scheduler/volumebinder"
```

Comment:
Can we move this to a group with the zap import and properly sort all the yunikorn imports and de-dupe applicationclient and appclient (keep using this one)?

Reply:
Imports ordering happens automatically with editor setup. Have taken care of de-dupe. Please check.

Reply:
I have just opened a new PR for the documentation. I had been sitting in my todo list for the last 9 months :-( Make sure you have the settings for the IDE as given in that apache/incubator-yunikorn-site#50. It will flag the imports as unsorted.